src/go/k8s/tests/e2e/pandaproxy-produce-consume-tls-client-external-ca/06-update-external-ca.yaml

```diff
@@ -0,0 +1,8 @@
+# Update the CA cert "provided" by the customer with self signed CA
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ca-cert
+data:
+  # Update the trusted CA to be self signed.
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURpRENDQW5BQ0NRRGV4WnhaelErK1JEQU5CZ2txaGtpRzl3MEJBUXNGQURDQmhURUxNQWtHQTFVRUJoTUMKVlZNeEN6QUpCZ05WQkFnTUFrbE1NUkF3RGdZRFZRUUhEQWREYUdsallXZHZNUll3RkFZRFZRUUtEQTF5WldSdwpZVzVrWVMxMFpYTjBNUTR3REFZRFZRUUxEQVZqYkc5MVpERU5NQXNHQTFVRUF3d0VkR1Z6ZERFZ01CNEdDU3FHClNJYjNEUUVKQVJZUmNHRjFiRUJ5WldSd1lXNWtZUzVqYjIwd0hoY05Nak13TXpJd01qQTFOek16V2hjTk16TXcKTXpFM01qQTFOek16V2pDQmhURUxNQWtHQTFVRUJoTUNWVk14Q3pBSkJnTlZCQWdNQWtsTU1SQXdEZ1lEVlFRSApEQWREYUdsallXZHZNUll3RkFZRFZRUUtEQTF5WldSd1lXNWtZUzEwWlhOME1RNHdEQVlEVlFRTERBVmpiRzkxClpERU5NQXNHQTFVRUF3d0VkR1Z6ZERFZ01CNEdDU3FHU0liM0RRRUpBUllSY0dGMWJFQnlaV1J3WVc1a1lTNWoKYjIwd2dnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUURBT2UvUS9XU1BaVldTUnQ3dgpEMlBsTlhEQUZma1NRNlk2Rlg0VXVmVForSytmcGl2VXpSUEh5RlBDMXl1Ujc2dXlURVlVQ3MwZUtPdmFrWWVPCmpUVzZaTDJ3V20xeU1XTkhpRGN5eng1SFByQmU5MnpiZHM1ZHl2elRtNEF3R2hCa1NGbldOeHZSemR2aEY1RzkKSlNFNXNFSzJqNG9GNDYxNjQzRFlKd0NaaFhxa0ZidXRubjlkSU5PSG1wWlJhYTZaZjB6ZDFwbzkveWQ5QlZNcQovcVh6QTdVa1k0WHBRVnNOWVc5NnFkeEhOYWx1ekZGRmdyRkhwTnhIbFZKKzNCK2IwM2sva2RLV0QzZDBxdVNDCkNsODN0YnRxT3FHZEEvdzV1ZU13OW5CZ0ZzRUgwOU5hVExoWFBSa0ZpbzF1Z2J1QVc3d1NsTUNhVGs4ek0zUnUKem40ekFnTUJBQUV3RFFZSktvWklodmNOQVFFTEJRQURnZ0VCQUpTblgvVkdqVlJuYnRLa2JkVlFib2NXa29tbwpqbXJXUyt6b0xmai8zaWdZbWJqSkREaDlpdzk3T0NLZ012OENUSm9RdmlPNUF6cE41R0ZvcFZBbWoyYzdDTzAvCis0Y3JVWmpMbkY1ZUlScTB3NGczVW9scTlpVmtSelRhbEVaNll3YXZ0R0hqWmY2TjZqVEt2Y3JFOFJFWFRHdUYKMm1EZ1ZoVS9kV1pQdFhadUZRKzVqMkdoaHFVaVQwN1FxbnF4NTBjbUN1WHowWHBBeGU5TUhyZ1l6YVJLRFZuawpZVGdFL0lzQllSMjc0RmI5MWFKRy9ITWd3STBxVmd6bWlFa1JWMmtYRG16Yk90eit3eThPcXFPNFVVZHF2Ylc3CjBYUm5iS3NDaTdUdUVVUXBTQVpTNnhERHJOVDZXZVBhU3BzclRCWTBjaDBYMTFjTytUVVQ3WllzSjZ3PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
You could just comment that this is private certificate comes from schema registry client.crt.

Reply:
Done.